### PR TITLE
Add hello world typescript template

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

File: repository.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+templates:
+  - name: Hello World Typescript
```
Comment (jyecusch) [inline repository.yaml]: Maybe the language name should be the prefix, e.g. "TypeScript - Hello World" to group language templates together in alphabetical output?

File: templates/hello-world-ts/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Typescript Hello World
```
Comment (jyecusch) [inline templates/hello-world-ts/README.md]: Tiny issue - TypeScript should have a capital S

**Review by jyecusch — CHANGES_REQUESTED** [review]

File: templates/hello-world-ts/package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "hello-world-ts",
+  "version": "1.0.0",
+  "description": "nitric typescript hello world template",
+  "main": "index.js",
+  "private": true,
+  "dependencies": {
+    "@nitric/sdk": "^0.3.0"
```
Comment (davemooreuws) [inline templates/hello-world-ts/package.json]: Perhaps change the version to "latest" to ensure we don't have to update this again.

**Review by davemooreuws — CHANGES_REQUESTED** [review]

File: repository.yaml (hunk 1 above)
Comment (tjholm) [reply repository.yaml]: fixed.

File: templates/hello-world-ts/README.md (hunk 2 above)
Comment (tjholm) [reply templates/hello-world-ts/README.md]: fixed.

File: templates/hello-world-ts/package.json (hunk 3 above)
Comment (tjholm) [reply templates/hello-world-ts/package.json]: fixed.

**Review by jyecusch — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**